src/engagement_db_to_analysis/traced_data_filters.py

```diff
@@ -60,15 +60,50 @@ def rqa_time_range_filter(user, messages_traced_data, pipeline_config):
         else:
             filtered.append(td)
 
-    log.info(f"Filtered out messages{time_range_log}. "
-             f"Returning {len(filtered)}/{len(messages_traced_data)} messages.")
+    log.info(f"Returning {len(filtered)}/{len(messages_traced_data)} messages...")
 
     return filtered
 
 
+def filter_test_participants(user, participants_traced_data_map, test_contacts):
+    """
+    Filters a dict of participants who are not in pipeline_config.test_contacts e.g AVF/Aggregator staff
+
+    :param participants_traced_data_map: A dict of TracedData individuals objects to filter.
+    :type participants_traced_data_map: dict of TracedData
+    :param test_contacts: a list containing test participant uids.
+    :type test_contacts: list of str
+    :return: Filtered dict.
+    :rtype: dict of participant_uid -> TracedData
+    """
+    log.debug("Filtering out test participants data...")
+    filtered = {}
+
+    for uid, trace_data in participants_traced_data_map.items():
```

Comment:
I think we should always say what the traced_data is e.g. use participant here (or possibly participant_traced_data), otherwise you always have to think about whether this refers to one traceddata history or to a list, and whether its messages or participants

Reply:
updated to participant_trace_data